CorsixTH/Src/th_map.cpp

```diff
@@ -249,12 +249,12 @@ map_tile_flags::operator uint32_t() const {
 }
 
 map_tile::map_tile()
-    : iParcelId(0),
+    : tile_layers{},
```

Comment:
As none of these depend on the constructor or the parameters they could be set on the members themselves. That would make ordering them more local.